src/controllers/HomeController.ts

```diff
@@ -1,6 +1,15 @@
 import { Request, Response, NextFunction } from 'express';
+import { controller, httpGet, BaseHttpController } from 'inversify-express-utils';
 
-export class HomeController {
+@controller('/')
+export class HomeController extends BaseHttpController {
 
-    public sayHello = (req: Request, res: Response, next: NextFunction) => res.render('index');
+    constructor() {
+        super();
+    }
+
+    @httpGet('index')
```

Comment:
IMO a bit to much is going on in that PR. It has DI, Redis and new page. I'd be cautious before adding more and more, we already have 15 files touched.

Reply:
this is more of a 'set up' ticket for redis and a playground for DI - that was my understanding. We plan to do the work for the ticket on a separate branch

Reply:
Good question, I am bit sad how this PR grows and by the fact extra bits like DI are included although I understand Francisco is working his way to establish some patterns.

Sadly it will grow even further when tests are added. I would not mind extracting some bits to smaller PR's if that simple and does not add unnecessary time.